config/flags_network.go

```diff
@@ -114,6 +114,12 @@ var (
 		Usage: "Range of UDP listen ports used for connections",
 		Value: "10000:60000",
 	}
+	// FlagTraversalOrder order of NAT traversal methods to be used for providing service.
+	FlagTraversal = cli.StringFlag{
+		Name:  "traversal",
+		Usage: "Comma separated order of NAT traversal methods to be used for providing service",
```

Comment:
Would it be useful to provide possible values here?

Reply:
we are including all values as default. Here is how it looks in help:
```
   --traversal value                                      Comma separated order of NAT traversal methods to be used for providing service (default: "holepunching,upnp,manual")
```